loader/loader.go

```diff
@@ -129,6 +132,21 @@ func (l *loader) GetEnterpriseContractPolicy(ctx context.Context, cli client.Cli
 	return enterpriseContractPolicy, getObject(releaseStrategy.Spec.Policy, releaseStrategy.Namespace, cli, ctx, enterpriseContractPolicy)
 }
 
+// GetEnterpriseContractConfigMap returns the defaults ConfigMap in the Enterprise Contract namespace . If the EC_CM_NAMESPACEDNAME
+// value is invalid or not set, nil is returned. If the ConfigMap is not found or the Get operation fails, an error is returned.
+func (l *loader) GetEnterpriseContractConfigMap(ctx context.Context, cli client.Client) (*corev1.ConfigMap, error) {
+	enterpriseContractConfigMap := &corev1.ConfigMap{}
+	namespacedName := os.Getenv("EC_CM_NAMESPACEDNAME")
```

Comment:
`ENTERPRISE_CONTRACT_CONFIG_MAP` sounds like a better name for the variable to me.